nix/trunk/default.nix

```diff
@@ -82,6 +95,7 @@ in {
           ++ (with pkgs.ocamlPackages; [hashcons tezt tezos-plompiler tezos-plonk pyml ppx_import ocaml-lsp]);
 
         shellHook = ''
+          export PATH=$PATH:${nodePackages}/lib/node_modules/.bin
```

Comment:
I assume this will break the installation of packages when in the repo? Can we reconcile these somehow?